hdl21/primitives.py

```diff
@@ -400,6 +405,25 @@ class PhysicalCapacitorParams:
     aliases=["Cap3", "PhyCap3", "CapPhy3", "PhyCapacitor3"],
 )
 
+# For the 4 terminal components, we have
```

Comment:
I don't think we should add this one as a generic primitive.
I don't see how it'll clearly generalize to other technologies.  
Recommend we keep it in the PDK / ExternalModules.